tests/drillhole_v4_0_test.py

```diff
@@ -611,11 +612,19 @@ def test_remove_drillhole_data(tmp_path):
         well = workspace.get_entity("well")[0]
         well_b = workspace.get_entity("Number 2")[0]
         data = well.get_data("my_log_values/")[0]
-        workspace.remove_entity(data)
-        workspace.remove_entity(well_b)
 
+        well.remove_children(data)
+        del data

```

Comment:
shouldn't we del the data in the remove option?

Reply:
This is to remove the variable within the context of the test at runtime. Otherwise the test itself has a reference to the object, and the garbage collection won't clean it out, and the weakref with survive.